<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>773</width>
    <height>567</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>60000</width>
    <height>60000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/bitbug_favicon.ico</normaloff>:/new/prefix1/bitbug_favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(85, 255, 127);</string>
   </property>
   <property name="lineWidth">
    <number>7</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>380</y>
     <width>194</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="OpenpushButton">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClosepushButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ClearpushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendpushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SendlineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>480</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(85, 255, 127);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear1pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>157</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>PortName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="portnamecomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Baud</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BaudcomboBox">
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>DataBits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DataBitscomboBox">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ParitycomboBox">
        <item>
         <property name="text">
          <string>无</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>奇校验</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>偶校验</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>StopBits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="StopBitscomboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>490</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="SendAsciiradioButton">
       <property name="text">
        <string>ASCII</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SendHexradioButton">
       <property name="text">
        <string>HEX</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="RecvAsciiradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="RecvHexradioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>110</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="TesttextEdit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>461</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(85, 255, 0);
</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="add_CRLF_checkBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>550</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>添加换行符</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
